.github/workflows/deploy.yml

```diff
@@ -30,13 +30,13 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: 16
-      - name: Install Yalc
-        run: sudo npm i yalc -g
-      - name: NPM Install
-        run: npm ci
+      - name: Checkout amazon-chime-sdk repo
+        uses: actions/checkout@v2
+        with:
+          repository: 'aws-samples/amazon-chime-sdk'
       - name: Deploy the Serverless Demo App
         run: |
-          cd $GITHUB_WORKSPACE/demo/meeting/serverless
+          cd apps/meeting/serverless
           node ./deploy.js -r us-east-1 -b chime-sdk-components-demo-canary -s chime-sdk-components-demo-canary -l
```

Comment:
This is actually not correct. You would need to pack the current changes in the current component repo (using npm pack) and do npm install that tar file in the meeting demo repo.
This will just deploy the latest npm release and not the github master.

Reply:
You may also want to update the js sdk version to the latest to match the dependency requirement in component library (basically the step we did in release script before).

Reply:
This is not addressed yet right?